apps/src/blockly/addons/cdoSerializationHelpers.js

```diff
@@ -0,0 +1,184 @@
+import BlockSvgUnused from './blockSvgUnused';
+import {WORKSPACE_PADDING} from '../constants';
+import {frameSizes} from './cdoConstants';
+
+const {HEADER_HEIGHT, MARGIN_BOTTOM, MARGIN_SIDE, MARGIN_TOP} = frameSizes;
+const SVG_FRAME_HEIGHT = HEADER_HEIGHT + MARGIN_TOP + MARGIN_BOTTOM;
+const SVG_FRAME_SIDE_PADDING = MARGIN_SIDE;
```

Comment:
Nit: maybe use `SvgFrame` object with values for `HEIGHT`, `SIDE_PADDING`, and `TOP_PADDING`.